src/QGCApplication.h

```diff
@@ -88,6 +88,9 @@ class QGCApplication : public QApplication
     void            setLanguage();
     QQuickWindow*   mainRootWindow();
     uint64_t        msecsSinceBoot(void) { return _msecsElapsedTime.elapsed(); }
+    QString         numberToString(quint64 number);
+    QString         bigSizeToString(quint64 size);
+    QString         storageFreeSizeToString(quint64 size_MB);
```

Comment:
I would rename this to be a bit more general:
```suggestion
    QString         bigSizeMBToString(quint64 size_MB);
```